### DISCUSSION TITLE
Are there any plans to support tree format?

### DISCUSSION THREAD

**Discussion by guaizi149** [samuelarbibe/dnd-timeline#52] [Ideas]
Tree structure data display, support asynchronous loading and Scroll Loading

**Comment by samuelarbibe** [comment]
Hey!  
All of these ideas will not be part of the core library, but rather individual implementations and usages of it.  
Integrating them into the core library is against the library's philosophy, which is being as generic and rudimentary as possible.  
  
If you need help in implementing one of these on your own, feel free to contact me.
